contents/docs/data-warehouse/start-here.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Getting started with data warehouse
+hideAnchor: true
+---
+
+import DWInstallationPlatforms from './_snippets/dw-installation-platforms'
+import { QuestLog, QuestLogItem } from 'components/Docs/QuestLog'
+import { IconGraph, IconRewindPlay, IconToggle  } from '@posthog/icons'
+
+
+<QuestLog firstSpeechBubble="Let's get started!" lastSpeechBubble="Time to build your data warehouse!">
+
+<QuestLogItem 
+    title="Link your first source"
+    subtitle="Required"
+    icon="IconCode2"
+>
+
+  Data warehouse lets you store, query, and analyze your data in a unified platform, so you can build better products and make data-driven decisions.
+
+  The first step is to link your first source to your data warehouse.
+
+  ### Platforms
+
+  <DWInstallationPlatforms />
+
+  <CallToAction type="primary" to="/docs/data-warehouse/sources">
```

Comment:
Is it always imported? You can use DW for data collected to PostHog too, right?

Or am I just confusing the sql editor with this.

Reply:
No, it's not always imported but I haven't fully figured out a way to talk about using PostHog data in the data warehouse, so it's probably best to start with importing data. 

You can use the data warehouse and SQL queries without setting up any sources if you want to.

Reply:
👍 Maybe there should be a data source that's just PostHog that we can add here 😆 And it just tells you to integrate posthog.

I think it's fine for now, but a little misleading when read out of context.